README.md

```diff
@@ -9,7 +9,7 @@ project.
 [![Crates.io](https://img.shields.io/crates/v/ratatui?logo=rust&style=flat-square)](https://crates.io/crates/ratatui)
 [![License](https://img.shields.io/crates/l/ratatui?style=flat-square)](./LICENSE) [![GitHub CI
 Status](https://img.shields.io/github/actions/workflow/status/ratatui-org/ratatui/ci.yml?style=flat-square&logo=github)](https://github.com/ratatui-org/ratatui/actions?query=workflow%3ACI+)
-[![Docs.rs](https://img.shields.io/docsrs/ratatui?logo=rust&style=flat-square)](https://docs.rs/crate/ratatui/)  
+[![Docs.rs](https://img.shields.io/docsrs/ratatui?logo=rust&style=flat-square)](https://docs.rs/crate/ratatui/)
```

Comment:
nit: trailing space is needed for breaking down the badges into two lines, but maybe there is a better way of doing it

Reply:
`\` works
https://github.github.com/gfm/#hard-line-break

Reply:
addressed in [63128f](https://github.com/ratatui-org/ratatui/pull/280/files#diff-63128fb186b5e73ba20dfb73d335955be10a51c76f6bf598efc1ada3e173e5b9)